server/oauth_service.py

```diff
@@ -0,0 +1,45 @@
+import aiocron
+import jwt
+
+from jwt import PyJWKClient, InvalidTokenError
+
+from server.config import config
+from .core import Service
+from .decorators import with_logger
+from .exceptions import AuthenticationError
+
+
+@with_logger
+class OauthService(Service):
+    """
+        Service for managing the OAuth token logins and verification.
```

Comment:
I decided I like this way better.
```suggestion
    Service for managing the OAuth token logins and verification.
```

Reply:
Done